.github/workflows/workflow-run-forge.yaml

```diff
@@ -112,6 +112,10 @@ jobs:
       - name: Install python deps
         run: pip3 install click==8.1.3 psutil==5.9.1
 
+      - name: Calculate Forge Auth Duration
+        id: calculate-auth-duration
+        run: echo "auth_duration=$(( $FORGE_RUNNER_DURATION_SECS + 600 ))" >> $GITHUB_OUTPUTS
```

Comment:
should we have some minimum? or add more than 600? 

5400 above is pretty long, this will potentially cause issues with some short tests that might not precisely respect the duration?

Reply:
just changed it to have a minimum of 90 min, and adds addl 30 min to the forge test runner length

Reply:
tested by canary:

https://github.com/aptos-labs/aptos-core/actions/runs/5359116756/jobs/9722293820, specifying runner duration 480s , defaults to 5400

<img width="448" alt="image" src="https://github.com/aptos-labs/aptos-core/assets/12578616/4f769918-925d-4021-aa0e-8d076c68416c">


https://github.com/aptos-labs/aptos-core/actions/runs/5359118196/jobs/9722297411, set runner duration to 5500, which is greater than the default 5400. 
<img width="467" alt="image" src="https://github.com/aptos-labs/aptos-core/assets/12578616/67cc8693-e94c-4934-b8b2-e30424bccf7b">